### DISCUSSION TITLE
License issues

### DISCUSSION THREAD

**Discussion by abi132** [aclap-dev/video-downloadhelper#1024] [License & Purchase requests]
### What's your issue?

I am having issues with the license I have proof of purchase and need to get my videos downloaded please help. if this issues continues I would like a refund as soon as possible!

### The 10 first digits (only 10) of your license number or email address

 f548858b38

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
> ### What's your issue?
> 
> I am having issues with the license I have proof of purchase and need to get my videos downloaded please help. if this issues continues I would like a refund as soon as possible!
> ### The 10 first digits (only 10) of your license number or email address
> 
> f548858b38
> ### I confirm that I did not not paste my full license number
> 
>     * [x]  I confirm

We need details Please tell us what is not working with your versions and a public link for the video if possible.
